1.3/spec.md

```diff
@@ -0,0 +1,180 @@
+# MBTiles 1.3
+
+## Abstract
+
+MBTiles is a specification for storing tiled map data in
+[SQLite](http://sqlite.org/) databases for immediate usage and for transfer.
```

Comment:
This isn't a wording change - I prefer this existing wording because it describes what the spec does.